Source/DafnyLanguageServer.Test/Various/SlowVerifier.cs

```diff
@@ -47,7 +43,7 @@ class NeverVerifiesImplementationTask : IImplementationTask {
       source = new TaskCompletionSource<VerificationResult>();
     }
 
-    public IObservable<VerificationStatus> ObservableStatus => Observable.Empty<VerificationStatus>();
+    public IObservable<VerificationStatus> ObservableStatus => Observable.Never<VerificationStatus>();
```

Comment:
Why are you leaving this empty observable open?

Reply:
Because this is the `NeverVerifiesImplementationTask`, so it shouldn't complete. It was making the tests HoverReturnsBeforeVerificationIsComplete and DefinitionReturnsBeforeVerificationIsComplete unstable.